editor/gui/scene_tree_editor.cpp

```diff
@@ -2373,6 +2380,14 @@ HashMap<Node *, SceneTreeEditor::CachedNode>::Iterator SceneTreeEditor::NodeCach
 	return I;
 }
 
+bool SceneTreeEditor::NodeCache::has(Node *p_node) {
+	HashMap<Node *, CachedNode>::Iterator I = get(p_node, false);
+	if (I) {
+		return true;
+	}
+	return false;
```

Comment:
```suggestion
	return get(p_node, false) != nullptr;
```

Reply:
a `HashMap<Node *, CachedNode>::Iterator` isn't a `nullptr` when the node isn't found. I am not entirely sure if this works reliably.

Reply:
I think explicitly calling `operator bool()` on the result is probably the way to go!